convert2rhel/utils.py

```diff
@@ -268,6 +268,16 @@ def get_traceback_str():
     """Get a traceback of an exception as a string."""
     exc_type, exc_value, exc_traceback = sys.exc_info()
     return "".join(traceback.format_exception(exc_type, exc_value, exc_traceback))
+                                              
+
+def remove_tmp_dir():
+    """Removes whole temporary folder /var/lib/convert2rhel/"""
+    try:
+        shutil.rmtree(TMP_DIR)
+        loggerinst.info("Temporary folder /var/lib/convert2rhel/ removed")
+    except OSError as err:
+        loggerinst.info("Failed removing backup folder /var/lib/convert2rhel/")
```

Comment:
If we change TMP_DIR the logger would be inaccurate, and if we did remember to change this we'd have more work for ourselves. Better to use the existing variable
```suggestion
        loggerinst.info("Temporary folder %s removed" % TMP_DIR)
    except OSError as err:
        loggerinst.info("Failed removing backup folder %s" % TMP_DIR)
```